reference/catalog/products_catalog.v3.yml

```diff
@@ -8252,6 +8252,8 @@ components:
           type: string
           description: |
             Type of gift-wrapping options. Values: `any` - allow any gift-wrapping options in the store; `none` - disallow gift-wrapping on the product; `list` – provide a list of IDs in the `gift_wrapping_options_list` field.
+
+            Always included in the response body; cannot exclude this field using the `exclude_fields` query parameter.
```

Comment:
Is it accurate that the fields are **always** included (since we can't use them for in the `exclude_fields` query parameter)?